app/src/main/res/layout/activity_signin.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why use a COnstraint layout yet the sub-layouts do not have any constraints attached?

Reply:
I thought I would use constraints at a future iteration. Lemme switch to relative